<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideShowdlg</class>
 <widget class="QDialog" name="SlideShowdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="7,1">
   <item>
    <widget class="QWidget" name="slideShow" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QWidget" name="slideprewid" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="PicButton" name="slidepreBtn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QWidget" name="slidenextwid" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="PicButton" name="slidenextBtn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="PicStateBtn" name="playBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PicButton" name="closeBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="PicAnimation" name="picanimation" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="preShow" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <widget class="PreListWid" name="preListWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>16777215</width>
           <height>110</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PicButton</class>
   <extends>QPushButton</extends>
   <header>picbutton.h</header>
  </customwidget>
  <customwidget>
   <class>PicStateBtn</class>
   <extends>QPushButton</extends>
   <header>picstatebtn.h</header>
  </customwidget>
  <customwidget>
   <class>PicAnimation</class>
   <extends>QWidget</extends>
   <header>picanimation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreListWid</class>
   <extends>QListWidget</extends>
   <header>prelistwid.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
